tests/fixtures/bootstrap_tests/config/env/post_update/test_post_update_new_parser.yml

```diff
@@ -7,7 +7,7 @@ frameworks:
   test_framework_v1.x.x:
     location:
       type: dev
-      path: '{CONFIG_FOLDER}/bundles/test_framework'
+      path: '{CONFIG_FOLDER}/bundles/test_framework_v1'
```

Comment:
The following changes are due to the fact that I renamed the test framework folder.